plugins/woocommerce/changelog/fix-41373

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Enforce product parent checks in variations REST API.
```

Comment:
Optional, as this is a bit subjective, but what do you think of:

```suggestion
Adds checks that ensure the product ID is valid when interacting with the product variations via the REST API.
```

(I'm just wondering if this or perhaps something else might help people reading the changelog to get a better sense of what changed—but, again, totally optional.)